client/state/login/test/selectors.js

```diff
@@ -28,10 +28,14 @@ import {
 	getSocialAccountLinkService,
 } from '../selectors';
 
+const EMPTY_STATE = {
+	login: {},
+};
+
 describe( 'selectors', () => {
 	describe( 'getTwoFactorUserId()', () => {
 		test( 'should return null if there is no information yet', () => {
-			const id = getTwoFactorUserId( undefined );
+			const id = getTwoFactorUserId( EMPTY_STATE );
 
 			expect( id ).to.be.null;
```

Comment:
If you don't mind more labor-intensive changes, it would be awesome to migrate these Chai asserts to Jest 🙂

Reply:
Done! 🙂